conjure-java-jaxrs-client/src/main/java/com/palantir/conjure/java/client/jaxrs/AbstractFeignJaxRsClientBuilder.java

```diff
@@ -119,13 +119,15 @@ private Contract createContract() {
     }
 
     private static Decoder createDecoder(ObjectMapper objectMapper, ObjectMapper cborObjectMapper) {
-        return new ConjureNeverReturnNullDecoder(
```

Comment:
I really don't want to move this because I think it's a really nice guarantee to say the conjure-java-runtime clients _can never return null_.  Part of our whole story about conjure is that you shouldn't have to think about the concept of `null` - we emphasise optionals instead.  I don't think we should reintroduce edgecases here.